### ISSUE TITLE
Function _deustlog always prints info logs

### ISSUE THREAD

**Issue by alexsm32** [ala15v/deust#9]
Function _deustlog always prints info logs. The function evaluates if the level is **_info_** or not but in both cases the action is ```env.info(fullmessage)```

```
if (level == 'info')
    then
        env.info(fullmessage)
    else
        env.info(fullmessage)
    end
```

**Comment by mac-developer** [comment]
@alexsm32 

It is supposed that it will be prefixed on log line:

https://github.com/ala15v/deust/blob/70cce104d4b76b1056d205d7da2158720052e203/development/deust/Debug/Debug.lua#L9

Do you refer to that?

**Comment by alexsm32** [comment]
Right, but a was pointing to the line 11 where the if else condition returns the same action in both cases.
